docs/project/hydro.md

```diff
@@ -14,11 +14,12 @@ LANG=zh . <(curl https://hydro.ac/setup.sh)
 
 使 Hydro 接入洛谷开放平台，您需要进行以下操作：
 
-1. 使用 `hydrooj install @hydrooj/vjudge && hydrooj install https://cdn.luogu.com.cn/lgapi/hydro/hydroplugin-luogu-0.0.1.tgz` 安装相关模块。
-2. 使用 `hydrooj cli luogu importProblem` 导入题库。
-3. 使用 `hydrooj cli luogu addAccount <token>` 设置账号。
-4. 重启 Hydro 进程（`pm2 restart hydrooj`）。
-5. 大功告成！
+1. 使用 `hydrooj install @hydrooj/vjudge` 安装 vjudge 插件（如果之前已安装请跳过该步骤） 
+2. 使用 `hydrooj install https://cdn.luogu.com.cn/lgapi/hydro/hydroplugin-luogu-0.0.1.tgz` 安装题库导入工具。
```

Comment:
```suggestion
2. 使用 `hydrooj install https://cdn.luogu.com.cn/lgapi/hydro/hydroplugin-luogu-latest.tgz` 安装题库导入工具。
```

Reply:
https://github.com/hydro-dev/Hydro/blob/master/packages/hydrooj/src/commands/install.ts#L48

Reply:
Now it supported.